### ISSUE TITLE
Separate domain for Auth Service, Gateway-MT from Linksharing

### ISSUE THREAD

**Issue by ferristocrat** [storj/gateway-mt#346]
### Goal

- Sometimes the entire storjshare.io domain will be blocked by anti-malware filters (such as Malwarebytes) preventing the use of linksharing as well as Storj hosted gateway-mt

### Acceptance Criteria

- New domain chosen for the exclusive use of Auth Service and Gateway-MT
- New domain implemented
- Existing domain still works (auth.storjshare.io, gateway.storjshare.io)

### Out of scope
- Updating documentation -- we'll determine if we want to update the docs separately
- Removing the use of storjshare.io

### Links

 - Relates to https://github.com/storj/gateway-mt/issues/320
